src/hooks/useFieldValidation.tsx

```
@@ -0,0 +1,78 @@
+    ).find((arr) => arr.length > 0);
+
+    const currentErrorType = currentErrorStatus?.[0];
+    return currentErrorType ? ERROR_TYPE_TO_MESSAGE[currentErrorType] : '';
+  }, [errorTypes]);
+
+  const isComplete = useMemo(() => {
+    return !Boolean(
+      (Object.values(inputValue) as string[]).filter(
+        (value) => value.length !== maxLength
+      ).length
+    );
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)